Makefile.kokkos

```diff
@@ -456,6 +463,14 @@ ifeq ($(KOKKOS_INTERNAL_USE_ARCH_AMD_GFX1100), 0)
   KOKKOS_INTERNAL_USE_ARCH_AMD_GFX1100 := $(call kokkos_has_string,$(KOKKOS_ARCH),NAVI1100)
 endif
 KOKKOS_INTERNAL_USE_ARCH_AMD_GFX1103 := $(call kokkos_has_string,$(KOKKOS_ARCH),AMD_GFX1103)
+KOKKOS_INTERNAL_USE_ARCH_AMD := $(shell expr $(KOKKOS_INTERNAL_USE_ARCH_AMD_GFX906)  \
+                                              + $(KOKKOS_INTERNAL_USE_ARCH_AMD_GFX908)  \
+                                              + $(KOKKOS_INTERNAL_USE_ARCH_AMD_GFX90A)  \
+                                              + $(KOKKOS_INTERNAL_USE_ARCH_AMD_GFX940)  \
+                                              + $(KOKKOS_INTERNAL_USE_ARCH_AMD_GFX942) \
+                                              + $(KOKKOS_INTERNAL_USE_ARCH_AMD_GFX1030) \
+                                              + $(KOKKOS_INTERNAL_USE_ARCH_AMD_GFX1100) \
+                                              + $(KOKKOS_INTERNAL_USE_ARCH_AMD_GFX1103))
```

Comment:
Should this be in a separate pull request?

Reply:
`KOKKOS_INTERNAL_USE_ARCH_AMD` is used by the OpenACC backend when targting AMD GPUs.